### PR TITLE
Specialist language

### PR DESCRIPTION
**Pull request by robredpath**

### CODE REVIEW

File: cove_360/templates/cove_360/explore.html [hunk 1]
```diff
@@ -219,9 +238,9 @@ <h4>Other</h4>
     <div class="panel-heading" data-toggle="collapse" data-target="#quality-accuracy-body">
       <h3 class="panel-title panel-title-explore">
         <span class="font-tick question" aria-hidden="true"></span>
-          {% trans "Quality / Accuracy" %}
```
Comment (BibianaC) [inline cove_360/templates/cove_360/explore.html]: I thought we were going to leave `accuracy` in the check name.

**Comment by BibianaC** [comment]
@robredpath some tests are failing.

File: cove_360/templates/cove_360/explore.html (hunk 1 above)
Comment (robredpath) [reply cove_360/templates/cove_360/explore.html]: @KDuerden 's feedback was that she wanted a single-word title, and I think I agree. I've also kept the internal naming of quality-accuracy as we might want to change that in the future!

**Comment by robredpath** [comment]
@BibianaC tests now pass :)

**Pull request merged**